build.cmd

```diff
@@ -0,0 +1,6 @@
+@echo off
```

Comment:
Thank you for the build script.

Please add the following capability to make the script more robust:

1) Detect the presence of VS tools, else bail out gracefully

2) Specify these in msbuild:
/p:Configuration
/l: logging when building